### PR TITLE
Add ability to run a single integration test

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

File: turborepo-tests/integration/turbo.json [hunk 1]
```diff
@@ -7,8 +7,12 @@
       "dependsOn": ["^topo"]
     },
     "test": {
-      "dependsOn": ["cli#build", "topo"],
-      "env": ["GO_TAG"]
+      "dependsOn": ["cli#build", "topo", "test:setup"]
```
Comment (chris-olszewski) [inline turborepo-tests/integration/turbo.json:10]: Is the dropping of `"env": ["GO_TAG"]` intentional?

**Review by chris-olszewski — APPROVED** [review]
Thanks for this!

File: turborepo-tests/integration/turbo.json (hunk 1 above)
Comment (mehulkar) [reply turborepo-tests/integration/turbo.json:10]: I thought it belonged in `cli#build` instead, and now that this dependency exist, that would cover our case 
https://github.com/vercel/turbo/blob/main/cli/turbo.json#L5-L6

**Comment by mehulkar** [comment]
Failed unit tests were due to time out, merging

**Pull request merged**